aiida/orm/nodes/data/cif.py

```diff
@@ -536,12 +556,14 @@ def get_formulae(self, mode='sum'):
         """
         # note: If formulae are not None, they could be returned
         # directly (but the function is very cheap anyhow).
-        formula_tag = '_chemical_formula_{}'.format(mode)
+        formula_tags = ('_chemical_formula_{}'.format(mode), '_pauling_file_chemical_formula')
```

Comment:
I am also not sure if we should build in support in such a hardcoded way for a tag that is custom to just one particular external database. I am not sure why they decided to have their own tag and if that respects the same spec as the official CIF `_chemica_formula_*` tag.

Reply:
There are (unfortunately) many of these "custom" CIF tags around. It's very good to include them and be flexible in this way, however, it should not be "hidden" within a general method like this, I think.
It would be good to dedicate special utility functions, dictionaries, methods or other to each _special_ list of tags, so they can be retrieved in this general method (or not).
Other than the `pauling` and `MPDS` specials, I know of `PDB` (Protein Data Bank) who have a very large number of special CIF tags (and even a dedicated format (PDBx/mmCIF), still using the `.cif` file type. These are listed in a special library/dictionary at [mmcif.wwpdb.org](http://mmcif.wwpdb.org/pdbx-mmcif-home-page.html).

Reply:
I guess one could identify the database from the CIF file and subsequently read the relevant tags, e.g. `_pauling_file_chemical_formula` in the case of MPDS.

Reply:
Where I think making the `parsing_formula` method more general is OK, since here we are talking strictly about parsing the CIF format, I would not hardcode these tags like this, because then we would potentially have to keep adding loads of them. How about, we add an optional argument `custom_tags=None` which can take a list of optional formula tags that should also be searched for?

Reply:
OK, this seems to be a good solution.